server.ts

```diff
@@ -60,6 +61,7 @@ app
   .use(ADFSAuthRoutes)
   .use(GoogleAuthRoutes)
   .use("/api", api)
+  .use("/api/admin", adminApi)
```

Comment:
another nitpick - if we're serving admin under /api, can we move the adminApi into the api router? 

ex. in server/api/index we add
router.use("/admin", adminApi)

Reply:
just makes more sense if they're both being served at /api/..
would love to hear your counter tho

Reply:
yep! I hadn't considered that we can do this, thanks for pointing it out!

Reply:
actually do we need a `/admin/` subdirectory? that might make things confusing later on. I have it removed right now, but I can re-add it if you object